docs/reference/rest-api.md

```diff
@@ -114,3 +114,60 @@ The formatting is based on the specified output format.
 
 On error, an "X-Stream-Error" header will be sent via the trailers channel with information about the error, and the stream will be closed via [`sender.abort()`](https://docs.rs/hyper/0.14.16/hyper/body/struct.Sender.html#method.abort).
 Depending on the client, the trailer header with error details may not be shown. The error will also be logged in quickwit ("Error when streaming search results").
+
+
+### Ingest data in an index
```

Comment:
Can we add something like "This endpoint is only available if you query on a node that is running an indexer service is running."?

Can you add such a comment also on the search endpoint? For the search though, it will work as long as you have one searcher in the cluster.